inference-engine/src/gna_plugin/layers/gna_split_layer.hpp

```diff
@@ -45,4 +45,18 @@ class GNASplitLayer {
     };
     std::vector<SplitConnectedLayerInfo> splitOutputLayers;
 };
+
+// @brief Returns sizes of split outputs to split the input tensor to aligned parts not greater than the specified size
+static std::vector<uint32_t> GetAlignedSplitSizes(uint32_t totalSize, uint32_t maxSplitSize, uint32_t alignment = 64) {
+    std::vector<uint32_t> splitSizes;
+    uint32_t maxAlignedSplitSize = maxSplitSize - maxSplitSize % alignment;
+    uint32_t usedSize = 0;
+    while (usedSize < totalSize) {
+        uint32_t partSize = std::min(totalSize - usedSize, maxAlignedSplitSize);
+        splitSizes.push_back(partSize);
+        usedSize += partSize;
+    }
+    return splitSizes;
```

Comment:
IE_ASSERT(usedSize == totalSize);?

Reply:
I think this check is extra since the algorithm can't produce usedSize != totalSize
- 'while' loop will break only when usedSize  >= totalSize
- on the next iteration usedSize will be increased with partSize which is equal to totalSize - usedSize, so it'll become equal to totalSize
So I don't see a way for this condition to be false in this algorithm.

Reply:
Don;t we need to cover this function by the new **unit** tests?

Reply:
unit test is added: gna_get_aligned_split_sizes.cpp